Documentation/observability/metrics.rst

```diff
@@ -546,8 +546,9 @@ Option Value          Description
 ===================== ===================================================================================
 ``identity``          All Cilium security identity labels
 ``namespace``         Kubernetes namespace name
-``pod``               Kubernetes pod name
+``pod``               Kubernetes pod name and namespace name in the form of ``namespace/pod``.
 ``pod-short``         Deprecated, will be removed in Cilium 1.14 - use ``workload-name|pod`` instead. Short version of the Kubernetes pod name. Typically the deployment/replicaset name.
```

Comment:
Can it then recommend `workload-name|pod-name`?

Reply:
Or even `app|workload-name|pod-name`? That seems like the closest to what Hubble does, no?

Reply:
> Or even app|workload-name|pod-name?
I'm not sure about that, it relies on users setting meaningful labels consistently, which is a bold assumption. workload-name and pod-name are given by Kubernetes.